autogpt_platform/backend/backend/blocks/discord/bot_blocks.py

```diff
@@ -1138,3 +1138,186 @@ async def run(
             raise ValueError(f"Login error occurred: {login_err}")
         except Exception as e:
             raise ValueError(f"An error occurred: {e}")
+
+
+class CreateDiscordThreadBlock(Block):
+    class Input(BlockSchema):
+        credentials: DiscordCredentials = DiscordCredentialsField()
+        channel_id: str = SchemaField(
```

Comment:
we generally allow this to be channel name or id and pair it with a server parameter